com.ktechpit.ultimate-media-downloader.yaml

```diff
@@ -0,0 +1,80 @@
+app-id: com.ktechpit.ultimate-media-downloader
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: ultimate-media-downloader
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --device=dri
+  - --socket=pulseaudio
+  
+tags:
+  - proprietary
+  
+modules:
+  - name: lzo
+    config-opts:
+      - '--enable-shared'
+      - '--disable-static'
+    cleanup:
+      - /include
+      - /share/doc
+      - '*.la'
+      - /lib/*.so
+    sources:
+      - type: archive
+        url: http://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
+        sha256: c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072
+
+  - name: squashfs-tools
+    buildsystem: simple
+    build-commands:
+      - XZ_SUPPORT=1 LZO_SUPPORT=1 make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=${FLATPAK_DEST}/bin
+    sources:
+    - type: git
+      url: https://github.com/plougher/squashfs-tools.git
+      tag: '4.5'
+      commit: '0496d7c3de3e09da37ba492081c86159806ebb07'
+      
+  - name: ultimate-media-downloader
+    buildsystem: simple
+    build-commands:
+      - install process_snap /
+      - install -Dm644 com.ktechpit.ultimate-media-downloader.metainfo.xml /share/metainfo/com.ktechpit.ultimate-media-downloader.metainfo.xml
+    sources:
+      - type: file
+        path: com.ktechpit.ultimate-media-downloader.metainfo.xml
+      - type: file
+        path: ultimate-media-downloader
```

Comment:
there is no such file.

Reply:
Fixed. The build is tested locally now. Everything works.